File: androidMvvm/app/src/main/java/com/mallcloud/androidmvvm/MainActivity.java
package com.mallcloud.androidmvvm;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import com.mallcloud.utils.DeadLockDemo;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    @Override
    protected void onResume() {
        super.onResume();
        DeadLockDemo.deadLock();
    }
}

File: androidkotlin/app/src/main/java/com/mallcloud/androidkotlin/MainActivity.kt
package com.mallcloud.androidkotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
//获取了引用的布局，比如  activity_main
import kotlinx.android.synthetic.main.activity_main.*

/**
 * kotlin首页
 */
class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        txtAllContent.setText("写个例子")

    }
}
